### DISCUSSION TITLE
People search

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#989] [2023 Attraction Contributions]
How do you find someone if they are not on social media?
I am looking for a very close friend, which is not in touch for some years now.
Any recommendation?

**Comment by hernessteve** [comment]
If you've exhausted other options and are still unable to locate your friend, you might consider hiring a private investigator. They have access to specialized resources and databases that can aid in finding missing persons.

**Comment by MirkosMangle** [comment]
Check out [MyHeritage reviews](https://myheritage.pissedconsumer.com/review.html). Maybe those guys will be able to help you.
I don't have a clear idea how does it work though. It is just a cheaper option than private investigator. 
That is another options of course.
